contrib/guix/libexec/build.sh

```diff
@@ -3,6 +3,14 @@ export LC_ALL=C
 set -e -o pipefail
 export TZ=UTC
 
+if [ -n "$V" ]; then
```

Comment:
No idea if it's a problem, but won't this detect V=0 as well?

Reply:
Right yes, I just wanted to conform with the `README` which says that we will do thing if "$V" is non-empty. Perhaps I can note in the `README` that we don't try to interpret the value of "$V" at all.